backend/experiment/models.py

```diff
@@ -9,16 +9,27 @@
 from theme.models import ThemeConfig
 from image.models import Image
 from session.models import Session
-from typing import Optional
+from typing import Optional, Union
 
 from .validators import markdown_html_validator, block_slug_validator
 
 language_choices = [(key, ISO_LANGUAGES[key]) for key in ISO_LANGUAGES.keys()]
 language_choices[0] = ("", "Unset")
 
 
-def consent_upload_path(instance, filename):
-    """Generate path to save consent file based on experiment.slug and language"""
+def consent_upload_path(instance: models.Model, filename: str) -> str:
+    """Generate path to save consent file based on experiment.slug and language
+    
+    Args:
+        instance (Experiment): Experiment instance to determine folder name
```

Comment:
Is the type `Experiment` or  `models.Model`? As stated above, remove the type definitions here and use the most specific type in the function definition.

Reply:
Good catch. `Experiment` wasn't recognized before, so I used `models.Model`. It does work with an import.